doc/data/messages/t/too-many-positional/bad.py

```diff
@@ -0,0 +1,6 @@
+class FiveArgumentMethods:
+    """The max positional arguments default is 5."""
+
+    # +1: [too-many-arguments, too-many-positional]]
+    def take_five_args(self, a, b, c, d, e):
```

Comment:
I welcome input on making this more user-friendly--I was just hammering this out quickly.